docs/guides/server/management-interface.adoc

```diff
@@ -0,0 +1,50 @@
+<#import "/templates/guide.adoc" as tmpl>
+<#import "/templates/kc.adoc" as kc>
+<#import "/templates/links.adoc" as links>
+
+<@tmpl.guide
+title="Configuring the Management Interface"
+summary="Learn how to configure {project_name}'s management interface for endpoints like metrics and health checks."
+includedOptions="http-management-* https-management-* legacy-observability-interface">
+
+The management interface allows accessing management endpoints via a different HTTP server than the primary one.
+It provides the possibility to hide endpoints like `/metrics` or `/health` from the outside world and, therefore, hardens the security.
+The most significant advantage might be seen in Kubernetes environments as the specific management port might not be exposed.
+
+== Management interface configuration
+
+The management interface is turned on by default, so management endpoints such as `/metrics`, and `/health` are exposed on the default management port `9000`.
+In order to change the port for the management interface, you can use the {project_name} option `http-management-port` or environment variable `KC_HTTP_MANAGEMENT_PORT`.
+
+The management interface provides a set of options and is fully configurable.
+If these options for the management HTTP server are not explicitly set, their values are automatically inherited from the default HTTP server.
+
+You can change the relative path of the management interface, as the prefix path for the management endpoints can be different.
+You can achieve it via the {project_name} option `http-management-relative-path` or environment variable `KC_HTTP_MANAGEMENT_RELATIVE_PATH`.
+
+For instance, if you set the CLI option `--http-management-relative-path=/management`, the metrics, and health endpoints will be accessed on the `/management/metrics` and `/management/health` paths.
```

Comment:
One thing we could add here, is that when a custom management path is not set, the custom context path via (--http-relative-path=auth) is still respected for the management endpoints, e.g. http://localhost:9000/auth/health and http://localhost:9000/auth/metrics.